exercises/C-variables/README.md

```diff
@@ -5,6 +5,8 @@ We can use _variable_ to create a reference to a value.
 ```js
 var greeting = "Hello world";
```

Comment:
The exercises use `var` a lot so it's understandable that you would also use `var`. We encourage you to use `let` or `const` instead and never `var`.

 - **let** - When a variable is assigned, its value can change
 - **const** - When a variable is assigned, its value will never change, it is a constant.
 
 We also don't need the \`\`\`js lines because this is more for markdown and will cause errors with your actual code.
 
 Finally we only need the `console.log(greeting);` one time otherwise you say Hello World three times 👍